src/domain/orders/details/swap/create.js

```diff
@@ -73,6 +80,36 @@ const extractPrice = (prices, order) => {
   return 0
 }
 
+const computeNewItemsTotal = (items, order) => {
+  const total = items.reduce(
+    (total, item) => total + extractPrice(item.prices, order) * item.quantity,
+    0
+  )
+  return `${total.toFixed(2)} ${order.currency_code.toUpperCase()}`
+}
+
+const prepareCustomShippingOptions = (
+  customShippingOptions,
+  shippingOptions
+) => {
+  if (!customShippingOptions.length) return []
```

Comment:
Flavor thing(but it's what i can comment on 😉 ), I think this should be in curly braces